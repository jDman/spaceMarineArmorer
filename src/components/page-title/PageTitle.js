import React, { Fragment } from 'react';

const PageTitle = (props) => {
  return (
    <Fragment>
      <h1>{props.title}</h1>
    </Fragment>
  );
};

export default PageTitle;
