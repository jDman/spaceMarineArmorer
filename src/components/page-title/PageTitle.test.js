import React from 'react';
import renderer from 'react-test-renderer';

import PageTitle from './PageTitle';

describe('<PageTitle />', () => {
  it('should display the title passed as a prop', () => {
    const title = 'TEST';

    const component = renderer.create(<PageTitle title={title} />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
