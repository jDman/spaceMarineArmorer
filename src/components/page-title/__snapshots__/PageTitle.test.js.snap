// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should display the title passed as a prop 1`] = `
<h1>
  TEST
</h1>
`;
